app/build.gradle.kts

```diff
@@ -62,9 +62,9 @@ android {
 dependencies {
 
 	// Required Dependencies to use the Zello SDK
-	implementation("com.zello:sdk:0.4.0")
-	implementation("com.zello:zello:0.4.0")
-	implementation("com.zello:core:0.4.0") {
+	implementation("com.zello:sdk:1.0.0")
```

Comment:
minor, but we could extract this into a variable so we only  have to change it once